kubetest/gke.go

```diff
@@ -487,15 +489,127 @@ func (g *gkeDeployer) TestSetup() error {
 	if err := g.ensureNat(); err != nil {
 		return err
 	}
+	if err := g.setupBastion(); err != nil {
+		return err
+	}
 	if err := g.setupEnv(); err != nil {
 		return err
 	}
-	if g.sshProxyInstanceName != "" {
-		if err := setKubeShhBastionEnv(g.project, g.zone, g.sshProxyInstanceName); err != nil {
-			return err
+	g.setup = true
```

Comment:
Why are we setting this now and weren't setting it before?

Reply:
just GUI diff error, it was there before as well